src/gui/Application.cpp

```diff
@@ -63,10 +63,20 @@ Application::Application(int& argc, char** argv)
     registerUnixSignals();
 #endif
 
+#if defined(Q_OS_WIN)
+    QString userName = qgetenv("USERNAME");
+    if (userName.isEmpty()) {
+        userName = qgetenv("USER");
+    }
+#else
     QString userName = qgetenv("USER");
     if (userName.isEmpty()) {
         userName = qgetenv("USERNAME");
     }
+#endif
+
+userName.replace('/', '_');
+userName.replace('\\', '_');
```

Comment:
This is insufficient, we should remove all characters that are not allowed in file names. https://stackoverflow.com/a/31976060